internals-js/src/definitions.ts

```diff
@@ -1072,8 +1099,50 @@ const graphQLBuiltInDirectivesSpecifications: readonly DirectiveSpecification[]
     locations: [DirectiveLocation.SCALAR],
     argumentFct: (schema) => ({ args: [{ name: 'url', type: new NonNullType(schema.stringType()) }], errors: [] })
   }),
+  // TODO: currently inconditionally adding @defer as the list of built-in. It's probably fine, but double check if we want to not do so when @defer-support is
```

Comment:
The general concept is that if `@defer` is "part of the schema" (visible via introspection if introspection is on, passes validation when included in an operation) then the server should "support defer" (though it could choose to not defer particular deferred fragments anyway).